core/src/Xios.cpp

```diff
@@ -116,10 +120,14 @@ void Xios::configureServer(const std::string calendarType)
 
     // Initialize calendar wrapper for 'nextSIM-DG' context
     cxios_get_current_calendar_wrapper(&clientCalendar);
-    cxios_set_calendar_wrapper_type(clientCalendar, calendarType.c_str(), calendarType.length());
-    cxios_duration timestep { 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0 };
-    cxios_set_calendar_wrapper_timestep(clientCalendar, timestep);
+    cxios_set_calendar_wrapper_type(clientCalendar, _calendarType.c_str(), _calendarType.length());
+    cxios_set_calendar_wrapper_timestep(clientCalendar, convertDurationToXios(_timestep));
     cxios_create_calendar(clientCalendar);
+    cxios_update_calendar_timestep(clientCalendar);
+
+    // Set default calendar origin and start // TODO: Pick something sensible
+    setCalendarOrigin(TimePoint("2020-01-23T00:08:15Z"));
+    setCalendarStart(TimePoint("2023-03-17T17:11:00Z"));
```

Comment:
Any thoughts on suitable default calendar origin and start times?

Reply:
I guess the start will be read from the config file.

Reply:
The time classes in the rest of nextSIM use the Unix epoch as their calendar origin.